polygeist/include/mlir/Dialect/Polygeist/Analysis/MemoryAccessAnalysis.h

```diff
@@ -456,6 +456,10 @@ class MemoryAccessAnalysis {
   std::optional<MemoryAccess>
   getMemoryAccess(const affine::MemRefAccess &access) const;
 
+  /// Return the grid dimension for \p funcOp (e.g the dimension of the
+  /// sycl::nditem or sycl::item argument passed to the to the function).
+  unsigned getGridSize(FunctionOpInterface funcOp) const;
```

Comment:
would getGridDimension be a better name, as size may suggest the number of threads in the whole grid.

Reply:
done